io.github.dyegoaurelio.simple-wireplumber-gui.json

```diff
@@ -0,0 +1,45 @@
+{
+  "app-id": "io.github.dyegoaurelio.simple-wireplumber-gui",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "44",
+  "sdk": "org.gnome.Sdk",
+  "command": "simple-wireplumber-gui",
+  "finish-args": [
+    "--socket=fallback-x11",
+    "--socket=wayland",
+    "--device=dri",
+    "--share=ipc",
+    "--filesystem=xdg-run/pipewire-0",
+    "--filesystem=~/.config/simple-wireplumber-gui/",
+    "--filesystem=~/.config/wireplumber/"
```

Comment:
why do you need this. it should the standard XDG locations.

Reply:
```python
CONFIG_PATH = f"/home/{current_user}/.config/simple-wireplumber-gui/config.json"
WIREPLUMBER_CONFIG_FOLDER = f"/home/{current_user}/.config/wireplumber"
```

That's even worse. It doesn't even honour `$HOME`.

Reply:
the `--filesystem=~/.config/wireplumber/` is needed because this app works by creating config files to wireplumber

the `--filesystem=~/.config/simple-wireplumber-gui/` is a mistake, I'll fix it

Reply:
@hfiguiere what's the best practice to define these paths ?